react/typings/ItemMetadata.d.ts

```diff
@@ -21,17 +21,31 @@ interface AssemblyOption {
   id: string
   name: string
   composition: Composition | null
-  inputValues: AttachmentField[]
+  inputValues: InputValue[]
 }
 
-interface AttachmentField {
+type InputValue = TextInputValue | BooleanInputValue | OptionsInputValue
+
+interface TextInputValue {
+  type: 'TEXT'|string
```

Comment:
Why do you need to define `type` as being `'TEXT' | string`, and not just `'TEXT'`? Other string values would be ok?

Reply:
Changed